src/components/Form/Form.jsx

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import Button from "../Button/Button";
+import DropdownField from "../DropdownField/DropdownField";
+import InputField from "../InputField/InputField";
+import "./Form.scss";
+
+const Form = ({
```

Comment:
Please call this something other than form as multiple forms are used and this isn't specific enough to this case. Same goes for file name.